app/models/event.rb

```diff
@@ -0,0 +1,6 @@
+class Event < ActiveRecord::Base
```

Comment:
This shortens the code but its a violation against rubocop, do we go by fewer lines that violate rubocop or many lines that that are okay i.e validate each param separately?